### ISSUE TITLE
Unable to Browse Phone Storage

### ISSUE THREAD

**Issue by atomspring** [nextcloud/android#2438]
### Actual behaviour
- Hello, I cannot seem to add files to upload, the list of available items is completely empty. This was fine on Android 7.1.2, I've recently upgraded to 8.1 on ARM64. Permissions are fine, something interesting in logcat though, it appears the "Invalid menu-item-ID" is consistently what appears when the list is empty (which it always is).

### Expected behaviour
- I expect to see a list of folders and files from my Internal Storage
 
### Steps to reproduce

1. I enable "Expert settings"
2. I Go to the "Auto Upload" area
3. "Create new custom folder setup"
4. Tap "Local Folder"
5. "No folders here", cannot see anything anywhere in the system

### Environment data
Android version: 8.1 x64

Device model:  Moto G5 (cedric)

Stock or customized system: customized

Nextcloud app version: 3.1.0 RC1

Nextcloud server version: 12.0.5

#### Pertinent Logcat
```04-07 09:21:28.767  3727  3727 V FileDisplayActivity: onPause() start
04-07 09:21:28.768  3727  3727 D FileDisplayActivity: onPause() ending
04-07 09:21:28.768  3727  3727 V FileDisplayActivity: onPause() end
04-07 09:21:28.805  3727  3727 D UploadFilesActivity: onCreate() start
04-07 09:21:28.807  3727  3727 D UploadFilesActivity: onCreate(Bundle) starting
04-07 09:21:28.818  3727  4200 D OwnCloudClient #7: REQUEST GET /ocs/v1.php/cloud/capabilities
04-07 09:21:28.827  3727  3727 I LocalFileListFragment: onCreateView() start
04-07 09:21:28.827  3727  3727 D ExtendedListFragment: onCreateView
04-07 09:21:28.947  3727  3727 I LocalFileListFragment: onCreateView() end
04-07 09:21:29.006  3727  3727 D UploadFilesActivity: onCreate() end
04-07 09:21:29.010  3727  3727 D UploadFilesActivity: onStart() starting
04-07 09:21:29.010  3727  3727 I LocalFileListFragment: onActivityCreated() start
04-07 09:21:29.018  3727  3727 I LocalFileListFragment: onActivityCreated() stop
04-07 09:21:29.043  3727  3727 D UploadFilesActivity: onResume() starting
04-07 09:21:29.044  3727  3727 W DrawerActivity: setDrawerMenuItemChecked has been called with invalid menu-item-ID
04-07 09:21:29.046  3727  4201 D ExternalLinks: links disabled
```

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky I can reproduce this issue and it also happens for manual file upload. It seems we jump in the file hierarchy at the wrong folder level.

**Comment by tobiasKaminsky** [comment]
@atomspring thank you for reporting this :+1: 
It seems that I overlooked it while implementing the new recycler view.
This will be in 3.1 :tada:

**Comment by AndyScherzinger** [comment]
PR has been merged for 3.1
